macro.pfsi/R/PfSI-Parameters.R

```diff
@@ -56,26 +58,30 @@ pfsi_parameters <- function(
     GSProtectPf = 1,
     gsBlockPf = .9,
     mnGSPf = 180,
-    vrGSPf = 20
+    vrGSPf = 20,
+    travel_vaxx = FALSE,
+    travel_treat = FALSE
   ){
   list(
-    Pf_c = Pf_c,
-    Pf_b = Pf_b,
-    DurationPf = DurationPf,
-    LatentPf = LatentPf,
-    FeverPf = FeverPf,
-    mnFeverPf = mnFeverPf,
-    vrFeverPf = vrFeverPf,
-    TreatPf = TreatPf,
-    mnTreatPf = mnTreatPf,
-    mnChemoprophylaxisPf = mnChemoprophylaxisPf,
-    PEProtectPf = PEProtectPf,
-    peBlockPf = peBlockPf,
-    mnPEPf = mnPEPf,
-    vrPEPf = vrPEPf,
-    GSProtectPf = GSProtectPf,
-    gsBlockPf = gsBlockPf,
-    mnGSPf = mnGSPf,
-    vrGSPf = vrGSPf
+    Pf_c = as.numeric(Pf_c),
+    Pf_b = as.numeric(Pf_b),
+    DurationPf = as.numeric(DurationPf),
+    LatentPf = as.numeric(LatentPf),
+    FeverPf = as.numeric(FeverPf),
+    mnFeverPf = as.numeric(mnFeverPf),
+    vrFeverPf = as.numeric(vrFeverPf),
+    TreatPf = as.numeric(TreatPf),
+    mnTreatPf = as.numeric(mnTreatPf),
+    mnChemoprophylaxisPf = as.numeric(mnChemoprophylaxisPf),
+    PEProtectPf = as.numeric(PEProtectPf),
+    peBlockPf = as.numeric(peBlockPf),
+    mnPEPf = as.numeric(mnPEPf),
+    vrPEPf = as.numeric(vrPEPf),
+    GSProtectPf = as.numeric(GSProtectPf),
+    gsBlockPf = as.numeric(gsBlockPf),
+    mnGSPf = as.numeric(mnGSPf),
+    vrGSPf = as.numeric(vrGSPf),
+    travel_vaxx = as.logical(travel_vaxx),
+    travel_treat = as.logical(travel_treat)
```

Comment:
This set of type checks, in this place, is a good idea. I'm starting to think it would help to make a class that associates a parameter name and value with its type.